src/style.css

```diff
@@ -0,0 +1,94 @@
+:root {
+  --padding-left-big-search: 62px;
+  --padding-left-small-search: 33px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  position: relative;
+  height: 70px;
+  margin-bottom: 20px;
+}
+
+.forms-wrapper {
+  margin-top: 20px;
+}
+
+.label-search-icon {
+  position: absolute;
+  top: 46%;
+  left: calc(var(--padding-left-big-search) / 2 + 1px);
+  transform: translateY(-30%) translateX(-30%);
+  cursor: text;
+}
+
+.icon {
+  width: 19px;
+  height: 19px;
+}
+
+.search {
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  height: 100%;
+  border-radius: 4px;
+  padding-left: var(--padding-left-big-search);
+  font-weight: 300;
+  font-size: 16px;
+  color: black;
+  font-family: Avenir, sans-serif;
+  margin-bottom: 20px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  box-shadow: 0 2px 2px gray;
+  outline: none;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.label-search-icon--small {
+  position: absolute;
+  left: calc(var(--padding-left-small-search) / 2);
+  top: 24%;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+}
+
+.search--small {
+  padding-left: var(--padding-left-small-search);
+  font-size: 14px;
+  background-size: 11px;
+  height: 42px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search--small:hover {
+  text-shadow: none;
+}
+
+.search--small:focus {
+  box-shadow: none;
+  text-shadow: none;
```

Comment:
Small input should have the same styles as a big input (normal state, focus, hover). You can create a common class for the common styles, and additional class for **search-small** where you will modify only the size of the element.

Reply:
Small search doesn't have shadow in Figma.